### PR TITLE
feat(v2 backend): make public landing redirect to appropriate app store.

### LINKED ISSUE #1970: v2 backend: app store redirect
## Summary

The v2 backend, when visited with a browser, has a placeholder `index.html` in Firebase Hosting. The user isn't redirected to the appropriate app store.

## Desired Solution

We'd like to set up our Firebase Hosting such that our `index.html` redirects to the appropriate app store page based on platform of the requester.

### PR DESCRIPTION
**Pull request by rjhuijsman**
Before this PR, the v2 backend hosted a default-generated `index.html` and `404.html` on Firebase Hosting. This PR updates these pages to redirect the visitor to the appropriate app store for their platform (using JavaScript), as well as providing useful content for those who have JavaScript disabled.

Closes #1970

## Screenshots

<details>
<summary>Screenshots</summary>

Landing page fallback (JS disabled), screen >=600px wide:
Screen Shot 2021-04-01 at 6 07 13 PM

Landing page fallback (JS disabled), screen <600px wide:
Screen Shot 2021-04-01 at 6 07 34 PM

</details>

## How did you test the change?

- [ ] iOS Simulator
- [ ] iOS Device
- [ ] Android Simulator
- [ ] Android Device
- [ ] `curl` to a dev App Engine server
- [X] other, please describe: ran `firebase serve --project=dev` and tested with Chrome Devtools.

## Checklist:

- [X] Followed the [Contributor Guidelines](https://github.com/WorldHealthOrganization/app/blob/master/docs/CONTRIBUTING.md) and verified that all contributions are properly licensed pursuant to the [LICENSE](https://github.com/WorldHealthOrganization/app/blob/master/LICENSE).

### CODE REVIEW

File: server/public/index.html [hunk 1]
```diff
@@ -1,89 +1,93 @@
...
-          ].filter(feature => typeof app[feature] === 'function');
-          loadEl.textContent = `Firebase SDK loaded with ${features.join(', ')}`;
-        } catch (e) {
-          console.error(e);
-          loadEl.textContent = 'Error loading the Firebase SDK, check the console.';
-        }
-      });
+      if (navigator.userAgent.toLowerCase().includes("android")) {
+        window.location = GOOGLE_PLAY_STORE_LINK;
+      } else {
```
Comment (bezfeng) [inline server/public/index.html:63]: Would it make sense if we checked for "iPad|iPhone|iPod" in the userAgent here to redirect to the App Store and showed the fallback page if we weren't sure what platform they're on?

**Review by bezfeng — APPROVED** [review]
Looks great! Just had some small questions but nothing blocking on my end

File: server/public/index.html [hunk 2]
```diff
@@ -1,89 +1,93 @@
...
-    <script defer src="/__/firebase/8.3.0/firebase-remote-config.js"></script>
-    <script defer src="/__/firebase/8.3.0/firebase-performance.js"></script>
-    <!-- 
-      initialize the SDK after all desired features are loaded, set useEmulator to false
-      to avoid connecting the SDK to running emulators.
-    -->
-    <script defer src="/__/firebase/init.js?useEmulator=true"></script>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <title>World Health Organization - Corona App</title>
```
[the remark this answers is not in the document]
Comment (matthewblain) [reply server/public/index.html:6]: Play store: "WHO COVID-19 Updates "
App on device "COVID-19".
Not sure if there's some string to pull from some other system/keep in sync/localize. Not real worried about that for now.

**Review by matthewblain — COMMENTED** [review]
i haven't reviwed the svg/html in detail (i'm sure it's ok) but a comment on the text.

File: server/public/404.html [hunk 3]
```diff
@@ -1,31 +1,21 @@
...
   <head>
-    <meta charset="utf-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
     <title>Page Not Found</title>
-
-    <style media="screen">
-      body { background: #ECEFF1; color: rgba(0,0,0,0.87); font-family: Roboto, Helvetica, Arial, sans-serif; margin: 0; padding: 0; }
-      #message { background: white; max-width: 360px; margin: 100px auto 16px; padding: 32px 24px 16px; border-radius: 3px; }
```
[the remark this answers is not in the document]
Comment (rjhuijsman) [reply server/public/404.html:10]: I considered it, but given that 99% of people will be redirected to the landing page anyway, I don't think it's worth much of an effort?

File: server/public/index.html [hunk 4]
```diff
@@ -1,89 +1,93 @@
...
-        // firebase.messaging().requestPermission().then(() => { });
-        // firebase.storage().ref('/path/to/ref').getDownloadURL().then(() => { });
-        // firebase.analytics(); // call to activate
-        // firebase.analytics().logEvent('tutorial_completed');
-        // firebase.performance(); // call to activate
-        //
-        // // 🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥
+      const GOOGLE_PLAY_STORE_LINK =
+        "https://play.google.com/store/apps/details?id=org.who.WHOMyHealth";
+      const APPLE_APP_STORE_LINK = "https://apps.apple.com/app/id1503458183";
```
[the remark this answers is not in the document]
Comment (rjhuijsman) [reply server/public/index.html]: That's great to know, thank you!

File: server/public/index.html (hunk 2 above)
[the remark this answers is not in the document]
Comment (rjhuijsman) [reply server/public/index.html:6]: Good call. I'll adjust to match the Play Store name.

File: server/public/index.html (hunk 1 above)
Comment (rjhuijsman) [reply server/public/index.html:63]: Good idea. Done.

**Review by brunobowden — APPROVED** [review]

**Pull request merged**